internal/backend/backend.go

```diff
@@ -348,10 +352,18 @@ func (o *Operation) Config() (*configs.Config, tfdiags.Diagnostics) {
 // more complex cases where e.g. diagnostics are interleaved with other
 // output, but terminating immediately after reporting error diagnostics is
 // common and can be expressed concisely via this method.
-func (o *Operation) ReportResult(op *RunningOperation, diags tfdiags.Diagnostics) {
+func (o *Operation) ReportResult(ctx context.Context, op *RunningOperation, diags tfdiags.Diagnostics) {
+	var span trace.Span
+	ctx, span = tracer.Start(ctx, "Operation.ReportResult")
```

Comment:
nitpick: I think we can simplify it just by writing it like this

```suggestion
	ctx, span := tracer.Start(ctx, "Operation.ReportResult")
```